src/Glpi/Form/AccessControl/FormAccessControl.php

```diff
@@ -284,6 +284,21 @@ public function getNormalizedInputName(string $name): string
         return "_access_control[{$this->getID()}][$name]";
     }
 
+    /**
+     * Verify that the given access control strategy is valid.
+     *
+     * @param string $strategy Strategy name (class name)
+     *
+     * @return bool
+     */
+    public function isValidStrategy(string $strategy): bool
+    {
+        return
+            is_a($strategy, ControlTypeInterface::class, true)
+            && !(new ReflectionClass($strategy))->isAbstract()
+        ;
+    }
+
```

Comment:
Useless change, probably due to a previous conflict.

Reply:
Visibility was changed to `public` as the method is now called from another file:

![image](https://github.com/user-attachments/assets/3e16d359-f031-4bda-8860-002955ca32d7)

Reply:
My bad. I thought it was just moved from a place to another in the file.